.github/workflows/tests.yml

```diff
@@ -0,0 +1,38 @@
+name: Camayoc Tests
```

Comment:
I think we might want to reconsider the name here. This action won't actually run the _camayoc_ test suite, but will just lint/do code coverage analysis on them.

Reply:
From the make file, it is about unit tests and test coverage for Camayoc itself."
`@echo "  test-coverage     to run unit tests and measure test coverage"`